src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorUtil.groovy

```diff
@@ -74,16 +71,17 @@ class IqPolicyEvaluatorUtil
 
       return evaluationResult
     }
-    catch (IqNetworkException e) {
+    catch (IqClientException e) {
```

Comment:
This is a different behaviour than we had before:
old: catches `Exception` and then searches the exception chain for `IOException`s.
new: only searches the exception chain if the exception is an `IqClientException`. 
So if there is an `IOException` wrapped in something else then `IqClientException` then the handler won't be called and the exception will probably be thrown to Jenkins.
Just wanted to check if this is intentional?

Reply:
Thanks, it was intentional but let me know if my logic breaks down. Rather than using rethrowNetworkErrors and IOException to scope which calls would constitute network errors. Since the rethrowNetworkErrors closure only surrounds calls to the nexus-java-api and the nexus-java-api wraps all network errors (and other checked exceptions) in IqClientException, I used IqClientException and IOException instead to filter the exceptions.